Corona-Warn-App/src/main/res/layout/include_expiration_notice_card.xml

```diff
@@ -11,7 +11,7 @@
 
     <TextView
         android:id="@+id/expiration_date_title"
-        style="@style/body2"
+        style="@style/subtitle"
         android:textStyle="bold"
```

Comment:
Maybe move `textStyle="bold"` into the style so we don't have to define it for every view 🤔. The problem ist that `subtitle` is used very often throughout the app, so this might introduce unwanted design changes ...